tests/components/lcn/test_init.py

```diff
@@ -144,13 +145,11 @@ async def test_migrate_1_1(hass: HomeAssistant, entry) -> None:
 
     entry_migrated = hass.config_entries.async_get_entry(entry_v1_1.entry_id)
     assert entry_migrated.state is ConfigEntryState.LOADED
-    assert entry_migrated.version == 2
-    assert entry_migrated.minor_version == 1
```

Comment:
With the migration, you want to explicitly test the versions after. Snapshots are not suitable for that.

Reply:
Does this only apply to the version numbers or should I also remove the snapshots for the `entity_migrated.data`?

Reply:
This is fine 👍 

Thanks for the changes. It explicitly checks if the migration took place and snapshots the data 👍